datasets/para_pat/para_pat.py

```diff
@@ -0,0 +1,226 @@
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(name="el-en", version=VERSION, description="This part of dataset covers el-en domain"),
+        datasets.BuilderConfig(name="cs-en", version=VERSION, description="This part of dataset covers cs-en domain"),
+        datasets.BuilderConfig(name="en-hu", version=VERSION, description="This part of dataset covers en-hu domain"),
+        datasets.BuilderConfig(name="en-ro", version=VERSION, description="This part of dataset covers en-ro domain"),
+        datasets.BuilderConfig(name="en-sk", version=VERSION, description="This part of dataset covers en-sk domain"),
+        datasets.BuilderConfig(name="en-uk", version=VERSION, description="This part of dataset covers en-uk domain"),
+        datasets.BuilderConfig(name="es-fr", version=VERSION, description="This part of dataset covers es-fr domain"),
+        datasets.BuilderConfig(name="fr-ru", version=VERSION, description="This part of dataset covers fr-ru domain"),
+        datasets.BuilderConfig(name="fr-ko", version=VERSION, description="This part of dataset covers fr-ko domain"),
+        datasets.BuilderConfig(name="ru-uk", version=VERSION, description="This part of dataset covers ru-uk domain"),
+        datasets.BuilderConfig(name="en-pt", version=VERSION, description="This part of dataset covers en-pt domain"),
+        datasets.BuilderConfig(name="de-fr", version=VERSION, description="This part of dataset covers de-fr domain"),
+        datasets.BuilderConfig(name="en-ja", version=VERSION, description="This part of dataset covers en-ja domain"),
+        datasets.BuilderConfig(name="en-es", version=VERSION, description="This part of dataset covers en-es domain"),
+        datasets.BuilderConfig(name="en-fr", version=VERSION, description="This part of dataset covers en-fr domain"),
+        datasets.BuilderConfig(name="de-en", version=VERSION, description="This part of dataset covers de-en domain"),
+        datasets.BuilderConfig(name="en-ko", version=VERSION, description="This part of dataset covers en-ko domain"),
+        datasets.BuilderConfig(name="fr-ja", version=VERSION, description="This part of dataset covers fr-ja domain"),
+        datasets.BuilderConfig(name="en-zh", version=VERSION, description="This part of dataset covers en-zh domain"),
+        datasets.BuilderConfig(name="en-ru", version=VERSION, description="This part of dataset covers en-ru domain"),
+    ]
+
+    def _info(self):
+        if self.config.name in type1_datasets_features:
+            features = datasets.Features(
+                {
+                    "index": datasets.Value("int32"),
+                    "family_id": datasets.Value("int32"),
+                    "src_lang": datasets.Value("string"),
+                    "src_abs": datasets.Value("string"),
+                    "tgt_lang": datasets.Value("string"),
+                    "tgt_abs": datasets.Value("string"),
```

Comment:
can you use the `Translation` feature type here ?
You can find an example of usage in [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py)